.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,49 @@
+name: Publish release
+
+on:
+  release:
+    types: created      
+  
+jobs:
+  publish-clinicarrivals:
+    runs-on: windows-latest
+    env:
+      buildtype: Release
+     
+    steps:
+    - name: Setup .NET Core SDK
+      uses: actions/setup-dotnet@v1.4.0
+    
+    - name: Setup nuget.exe
+      uses: warrenbuckley/Setup-Nuget@v1
+    
+    - name: Setup msbuild.exe
+      uses: microsoft/setup-msbuild@v1
+
+    - name: Checkout source code
+      uses: actions/checkout@v2.0.0
+  
+    - name: Restore packages
+      run: nuget restore
+    
+    - name: Build ClinicArrivals
+      run: msbuild /p:Configuration=${{env.buildtype}} -maxcpucount:2 ClinicArrivals.sln
+    
+    - name: Upload artifact
+      uses: actions/upload-artifact@v1.0.0
+      with:
+        name: ClinicArrivals
+        path: ${{runner.workspace}}\ClinicArrivals\ClinicArrivals\bin\${{env.buildtype}}
+            
+    - name: Create release archive
+      run: 7z a ClinArrivals-${{github.event.release.tag_name}}-win.zip ${{runner.workspace}}\ClinicArrivals\ClinicArrivals\bin\${{env.buildtype}}\*
+                
+    - name: Upload archive to release
+      uses: actions/upload-release-asset@v1.0.1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ github.event.release.upload_url }}
+        asset_name: ClinArrivals-${{github.event.release.tag_name}}-win.zip
+        asset_path: ClinArrivals-${{github.event.release.tag_name}}-win.zip
```

Comment:
Did you say ClinArrivals because of a length limitation? If not, can we say ClinicArrivals?

Reply:
Fixed!